src/Core/ViewHelper/AbstractViewHelper.php

```diff
@@ -146,6 +146,18 @@ public function isOutputEscapingEnabled()
         return $this->escapeOutput !== false;
     }
 
+    /**
+     * Returns the name of variable that contains the value to use instead of render children closure, if specified.
+     * ViewHelpers that want to use contentArgumentName are expected to override this method with their own implementation.
+     *
+     * @return string|null
+     * @api
+     */
+    public function resolveContentArgumentName()
```

Comment:
If this needs to be resolved or not shouldn't matter to the API. Thus `getContentArgumentName` would be more appropriate.

Reply:
I thought a bit about it. While migrating ViewHelpers is a bit more effort now, on the whole it's best to introduce a better name (that can then also use a type hint, which wouldn't be possible with the old name).